app/assets/stylesheets/devise_areas.scss

```diff
@@ -0,0 +1,21 @@
+.shared-links {
+  float: right;
+  padding: 7px;
+}
+
+.shared-links a {
+  font-size: 1.5rem;
+}
+
+.shared-links a:hover {
+  color: #D4827D;
```

Comment:
Color literals like `#D4827D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#D4827D` should be written as `#d4827d`